news/2019-11-11-winter-2019-fanart-contest.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+title: "Winter 2019 Fanart Contest"
+date: 2019-11-11 18:00:00 +0000
+---
+
+The end of the year is rapidly approaching, and while things might cooling down where you are (or maybe not), the artist's battlefield is still just as firey as ever!
+
+![](https://assets.ppy.sh/contests/82/header.jpg)
+
+Do you like to draw? Great, because we've got just the contest for you!
+
+**Draw any of the osu! mascots (official or community) in a winter themed piece** and you could have a shot at getting your work showcased in the game's main menu as a part of our Winter 2019 Seasonal Background pack!
+
+The hard-fought finalists will also nab themselves a few months of **osu!supporter** (the exact amount yet to be determined), and the top three finalists as voted by YOU will receive an extra shiny profile badge showcasing their work!
+
+But Ephemeral, you might ask, these are *new* prizes, what gives? What gives is the season of giving, and that's precisely what this time of year tends to be for most of the world. That and most importantly, we've heard your feedback on the prizes for the winning artists crystal clear. *(Yes, previous fanart contest top 3 finalists will eventually receive profile badges for their winning entries as well. You'll not be left out!)*
+
+Think you're up for the challenge? You've got **20** days from RIGHT NOW to get your concept onto paper and into your computer, then in to our contest system. As always, the community will be forced to make many impossible choices with far too many votes to help us decide which ones will be our final entries.
+
+Make sure you read the rules below before you enter - especially if you've done this before, since there's a few new ones this time around!
+
+## Rules
+
+* **All entries MUST be winter themed and feature one (or more) of the osu! mascots (either official or community).** Cold, Christmas, mountains or whatever else, just make sure that it is really obviously winter at a glance for most of the world and that a mascot we know and love features in it. Any lonesome Dons will find themselves figuratively and literally out in the cold this year.
+* **All entries must be suitable for all-ages viewing.** Our mascots are pure and innocent and would never be wrapped up in nothing but ribbons and put under a festive tree. Someone might have done this a few years back and we had to pull their entry because of it - YOU KNOW WHO YOU ARE! Keep it clean.
```

Comment:
```suggestion
* **All entries must be suitable for all-ages viewing.** Our mascots are pure and innocent and would never be wrapped up in nothing but ribbons and put under a festive tree. Someone might have done this a few years back and we had to pull their entry because of it — YOU KNOW WHO YOU ARE! Keep it clean.
```
grand compromise